app/src/main/java/org/oppia/android/app/help/LoadFAQListFragmentListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.android.app.help
+
+/** Listener for when a selection should result to [FAQListFragment] in tablet devices. */
+interface LoadFAQListFragmentListener {
```

Comment:
```suggestion
interface LoadFaqListFragmentListener {
```

Per https://developer.android.com/kotlin/style-guide#camel_case. (Note that many existing FAQ components have incorrect names).

Reply:
Done.